src/styles/scss/figures.scss

```diff
@@ -1,14 +1,18 @@
 .figure-wrapper {
+  display: flex;
+  flex-wrap: wrap;
   margin: 0 -6px;
 }
 
 .figure {
   background: $gray-lighterest;
   border: 1px solid $gray-lighter;
   border-radius: 3px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
```

Comment:
I think the figures in the Runnabear section look nice with `justify-content: flex-end`, but it doesn't look as nice for the Icon section. I think you should put that property specifically on `.figure-bears`.

Also, it looks like something is causing the images inthe Icon section to have extra space on the top and bottom. You may need to give these images a height (right now it looks like they only have a width specified).

![image](https://cloud.githubusercontent.com/assets/7440805/24595702/02da83b0-17ee-11e7-9723-63eb495835e9.png)